models/audio/train_inception.py

```diff
@@ -0,0 +1,96 @@
+from keras.models import Sequential, Model
+from keras.layers import Input, Dense, TimeDistributed, LSTM, Dropout, Activation, ELU, ReLU
+from keras.layers import Convolution2D, MaxPooling2D, Flatten, GlobalAveragePooling2D
+from keras.utils import image_dataset_from_directory
+from keras.applications import InceptionV3
+from keras.optimizers import Adam
+from keras.regularizers import L2
+from keras.callbacks import ModelCheckpoint
+import matplotlib.pyplot as plt
+import tensorflow as tf
+import os
+
+
+def define_InceptionV3_model(input_shape, nb_classes):
+    conv_base = InceptionV3(include_top=False, weights='imagenet', input_shape=input_shape)
+    print("input shape", input_shape)
+    model = Sequential()
+    model.add(conv_base)
+    model.add(Flatten())
+    model.add(Dense(512, name='dense_1', kernel_regularizer=L2(0.001)))
+    model.add(Dropout(rate=0.5, name='dropout_1'))
+    model.add(Dense(256, name='dense_2', kernel_regularizer=L2(0.001)))
+    model.add(Dropout(rate=0.3, name='dropout_2'))
+    model.add(Activation(activation='relu', name='activation_1'))
+    model.add(Dense(nb_classes, activation='softmax', name='dense_output'))
+    conv_base.trainable = False
+    return model
+
+
+def process(image, label):
+    image = tf.cast(image / 255., tf.float32)
+    return image, label
+
+
+def train_val_split(path, batch_size, img_shape):
```

Comment:
Can you define this function as tool for testing all models? It should be similar for any type of NN.

Reply:
Good idea